tsconfig.json

```diff
@@ -46,7 +46,11 @@
       "packages/opentelemetry-semantic-conventions",
       "packages/opentelemetry-shim-opentracing",
       "packages/propagator-aws-xray",
-      "packages/sdk-metrics"
+      "packages/sdk-metrics",
+      "integrated-benchmarks/http",
+      "integrated-benchmarks/http-otel",
+      "integrated-benchmarks/express",
+      "integrated-benchmarks/express-otel"
```

Comment:
@pichlermarc do you know why these get removed on `npm i` are they meant to be somewhere else?

Reply:
I think these may also need to be added to the npm workspace and and then they should stay even when `npm install`ing. Same as below.